callstack/react-native-visionos
# chore: add warning when using `InputAccessoryView`

## Summary:

Closing #8 
Since `InputAccessoryView` is not supported on VisionOS, this change adds a warning message about the component being unavailable on the platform.



## Changelog:
[VISIONOS] [ADDED] - warning about unavailable `InputAccessoryView`


## Test Plan:
1. Open `InputAccessoryView` in RNTester
2. Verify warn is displayed in the app and in the Metro logs

<img width="468" alt="image" src="https://github.com/callstack/react-native-visionos/assets/13985840/c90ff855-3b0c-4918-a49d-4d9c7aa37b58">

<img width="1016" alt="Screenshot 2023-10-28 at 11 38 48" src="https://github.com/callstack/react-native-visionos/assets/13985840/818e5cde-f68a-4f1c-a4bf-d844a786ecfa">
